tests/manage/z_cluster/cluster_expansion/test_node_expansion.py

```diff
@@ -6,12 +6,14 @@
     skipif_managed_service,
     skipif_flexy_deployment,
     skipif_ibm_flash,
+    skipif_ms_consumer,
```

Comment:
For the test to run on provider you'll need to remove skipif_managed_service mark, and for the test below as well.

Reply:
Yes I am thinking about how to accommodate both so that it should accommodate skip on rosa/osd tier1 for converged mode and v2 managed service solution as well